README.md

```diff
@@ -22,6 +22,7 @@ repositories {
 modularAssertionBuilder {
     gatewayBaseVersion = '<gateway-version>'
     assertionName='My-Modular-Assertion'
+    revision='<source code revision id>'
```

Comment:
Maybe add more details on what revision is. Is it supposed to be a build number?